From fc2d1b58c4f2c7240093d738ca99cfcf7a8b3107 Mon Sep 17 00:00:00 2001
From: Namjae Jeon <namjae.jeon@samsung.com>
Date: Wed, 26 May 2021 18:01:08 +0900
Subject: [PATCH] cifsd: Prefer kernel type 'u16' over 'uint16_t'
Git-commit: fc2d1b58c4f2c7240093d738ca99cfcf7a8b3107
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


Fix a warning from checkpatch.pl --strict:

 CHECK: Prefer kernel type 'u16' over 'uint16_t'
 #112: FILE: server.c:112:
 +       uint16_t command;

Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/cifsd/server.c     | 4 ++--
 fs/cifsd/smb2pdu.c    | 2 +-
 fs/cifsd/smb_common.h | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/fs/cifsd/server.c b/fs/cifsd/server.c
index 4ba43e788ce9..a99963b849d5 100644
--- a/fs/cifsd/server.c
+++ b/fs/cifsd/server.c
@@ -106,10 +106,10 @@ static inline int check_conn_state(struct ksmbd_work *work)
 #define TCP_HANDLER_ABORT	1
 
 static int __process_request(struct ksmbd_work *work, struct ksmbd_conn *conn,
-			     uint16_t *cmd)
+			     u16 *cmd)
 {
 	struct smb_version_cmds *cmds;
-	uint16_t command;
+	u16 command;
 	int ret;
 
 	if (check_conn_state(work))
diff --git a/fs/cifsd/smb2pdu.c b/fs/cifsd/smb2pdu.c
index 84b243b3895a..212cdffd27bc 100644
--- a/fs/cifsd/smb2pdu.c
+++ b/fs/cifsd/smb2pdu.c
@@ -193,7 +193,7 @@ int is_smb2_rsp(struct ksmbd_work *work)
  *
  * Return:      smb2 request command value
  */
-uint16_t get_smb2_cmd_val(struct ksmbd_work *work)
+u16 get_smb2_cmd_val(struct ksmbd_work *work)
 {
 	struct smb2_hdr *rcv_hdr;
 
diff --git a/fs/cifsd/smb_common.h b/fs/cifsd/smb_common.h
index 2d7b1c693ff4..6e7404b8db96 100644
--- a/fs/cifsd/smb_common.h
+++ b/fs/cifsd/smb_common.h
@@ -469,7 +469,7 @@ struct filesystem_posix_info {
 } __packed;
 
 struct smb_version_ops {
-	uint16_t (*get_cmd_val)(struct ksmbd_work *swork);
+	u16 (*get_cmd_val)(struct ksmbd_work *swork);
 	int (*init_rsp_hdr)(struct ksmbd_work *swork);
 	void (*set_rsp_status)(struct ksmbd_work *swork, __le32 err);
 	int (*allocate_rsp_buf)(struct ksmbd_work *work);
-- 
2.33.0

